dsl-reference.md

```diff
@@ -1416,6 +1417,36 @@ from:
 to: '.petList += [ . ]'
 ```
 
+### Export
+
+Certain task needs to set the context of the workflow using the task output for later usage. User set the content of the context through a runtime expression. The result of the expression is the new value of the context. If user want to merge the new data into the current context, the context is accessible within the expression scope: `.` if using jq. 
+
+Optionally, the context can have a predefined schema. 
+
+
+#### Properties
+
+| Property | Type | Required | Description |
+|----------|:----:|:--------:|-------------|
+| schema | [`schema`](#schema) | `no` | The [`schema`](#schema) used to describe and validate context.<br>*Included to handle the non frequent case in which the context has a known format.* |
+| as | `string`<br>`object` | `no` | A [runtime expression](#runtime-expressions), if any, used to set the context value. |
+
+#### Examples
+
+Assuming the output of the task is a json object (not primitive, not array)
```

Comment:
Very good point. The workflow input and the context data are the only dynamic data that are REQUIRED to be a mapping.

However the context could have nothing to do with JSON. You could write/read it from any format. 

I would therefore reformulate the phrase like this, to keep the same linguo than in other places of the document:

```md
Assuming the output of the task is a mapping
```